src/main/kotlin/io/bazel/kotlin/plugin/jdeps/BUILD.bazel

```diff
@@ -25,6 +25,7 @@ kt_bootstrap_library(
         "//src/main/kotlin/io/bazel/kotlin/builder/utils/jars",
         "//src/main/protobuf:deps_java_proto",
         "@com_github_jetbrains_kotlin//:kotlin-compiler",
+        "@kotlin_rules_maven//:com_google_guava_guava",
```

Comment:
How much does this increase the release binary size?

Reply:
Final `rules_kotlin_release.tgz` went 19Mb->21.8Mb